src/supermin-init-prelude.sh

```diff
@@ -23,7 +23,7 @@ LANG=C /sbin/load_policy  -i
 
 # we want /dev/disk symlinks for coreos-installer
 /usr/lib/systemd/systemd-udevd --daemon
-/usr/sbin/udevadm trigger --settle
+timeout 30s /usr/sbin/udevadm trigger --settle || :
```

Comment:
might be nice here to explain in the comment above why we have the timeout.

Reply:
Added a comment!